epam-promo/components/pickers/DataPickerBody.tsx

```diff
@@ -23,9 +24,24 @@ export class DataPickerBody extends PickerBodyBase<DataPickerBodyProps> {
             return this.props.renderNotFound();
         }
 
+        const renderDefaultBody =  () => {
```

Comment:
let create error block directly into modal and input components and pass them via `props.renderNotFound` to DataPickerBody